crates/host/src/wasmbus/mod.rs

```diff
@@ -3218,6 +3113,33 @@ fn human_friendly_uptime(uptime: Duration) -> String {
     .to_string()
 }
 
+fn to_core_entity(
```

Comment:
Looking at the name of this function, its purpose is not clear

Reply:
I agree, I'm not happy with the name. Do you have an idea for a better name?